run-tests.pl

```diff
@@ -470,6 +491,15 @@ sub _run_test
       1;
    };
 
+   Future->needs_all( map {
+      if( is_Fixture( $_ ) and $_->teardown ) {
```

Comment:
It does. It calls the `teardown` accessor method on the `Fixture` structure. That just returns the value of that structure field; which is itself a `CODE` reference. It doesn't invoke the actual function referred by that.